searx/templates/oscar/stats.html

```diff
@@ -92,6 +92,126 @@ <h1>{{ _('Engine stats') }}</h1>
+                        <table class="table table-striped table-bordered">
+                            <tbody style="padding-top: 1rem;">
+                                <tr>
+                                    {% if error.exception_classname %}
+                                        <th scope="row" style="width: 10rem">{{ _('Exception') }}</th><td>{{ error.exception_classname }}</td>
+                                    {% elif error.log_message %}
+                                        <th scope="row" style="width: 10rem">{{ _('Message') }}</th><td>{{ error.log_message }}</td>
+                                    {% endif %}
+                                    <th scope="row" style="width: 10rem">{{ _('Percentage') }}</th><td style="width: 10rem">{{ error.percentage }}</td>
+                                </tr>
+                                {% if error.log_parameters and error.log_parameters != (None, None, None) %}<tr><th scope="row">{{ _('Parameter') }}</th>
+                                    <td colspan="3">
+                                        {% for param in error.log_parameters %}
+                                            <span style="border-right: 1px solid gray; padding: 0 1rem 0 0; margin: 0 0 0 0.5rem;">{{ param }}</span>
+                                        {% endfor %}
+                                    </td>
+                                </tr>{% endif %}
+                                <tr><th scope="row">{{ _('Filename') }}</th><td colspan="3">{{ error.filename }}:{{ error.line_no }}</td></tr>
+                                <tr><th scope="row">{{ _('Function') }}</th><td colspan="3">{{ error.function }}</td></tr>
+                                <tr><th scope="row">{{ _('Code') }}</th><td colspan="3">{{ error.code }}</td></tr>
+                            </tbody>
+                        </table>
+                    {% endif %}
+                {% endfor %}
+            {% endfor %}
+            {% if engine_reliabilities[selected_engine_name].checker %}
+                <h3>{{ _('Checker') }}</h3>
+                <table class="table table-striped table-bordered">
+                    <tr>
+                        <th scope="col" style="width: 10rem">{{ _('Failed test') }}</th>
+                        <th scope="col">{{ _('Comment(s)') }}</th>
+                    </tr>
+                    {% for test_name, results in engine_reliabilities[selected_engine_name].checker.items() %}
+                    <tr>
+                        <td>{{ test_name }}</td>
+                        <td>
+                            {% for r in results %}<p>{{ r }}</p>{% endfor %}
+                        </td>
+                    </tr>
+                    {% endfor %}
+                </table>
+            {% endif %}
+            <form action="https://github.com/searxng/searxng/issues/new" method="GET">
+                <input name="title" type="hidden" value="Issue about {{ selected_engine_name }}">
```

Comment:
`https://github.com/searxng/searxng/issues/new` could be replaced by `"{{ brand.GIT_URL }}/issues/new"`

Reply:
In the final version we should have a form for the new issue in ``searx/templates/__common__``. I also suggest to define a ``{{ brand.NEW_ISSUE_URL }}``